apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/pagerules/PageRuleController.java

```diff
@@ -65,4 +68,10 @@ public CreateRuleResponse updatePageRule(@PathVariable Long ruleId,
     public ViewRuleResponse viewRuleResponse(@PathVariable Long ruleId) {
         return pageRuleService.getRuleResponse(ruleId);
     }
+    @PreAuthorize("hasAuthority('LDFADMINISTRATION-SYSTEM')")
+    @GetMapping("rule")
+    @ResponseBody
+    public Page<ViewRuleResponse> getAllPageRule(@PageableDefault(size = 25) Pageable pageable){
+        return pageRuleService.getAllPageRule(pageable);
+    }
```

Comment:
While you are making changes to the controller, would you mind updating the route to be `rules` instead of `rule`. 
This could be moved into the mapping on line 21 and removed from all of the other routes.

We could also move the `@PreAuthorize` to the class level and remove it from each endpoint.